std/algorithm/comparison.d

```diff
@@ -738,7 +741,7 @@ template equal(alias pred = "a == b")
             if (len1 != len2) return false; //Short circuit return
 
             //Lengths are the same, so we need to do an actual comparison
-            //Good news is we can sqeeze out a bit of performance by not checking if r2 is empty
+            //Good news is we can squeeze out a bit of performance by not checking if r2 is empty
```

Comment:
Why is this change here? This should be a separate PR, to avoid mixing different kinds of changes into a single PR.